### PR TITLE
Added Github Action Continuous Integration to the repo.

### PR DESCRIPTION
**Pull request by ggfan**
Some example of github Action CI could be found here:
   https://github.com/google/oboe/actions
PTAL

### CODE REVIEW

**Review by philburk — APPROVED** [review]
Looks great. Let's merge this and then remove the Travis CI.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: build samples...
```
Comment (philburk) [inline .github/workflows/build.yml:21]: The ... could be a problem

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
```
Comment (philburk) [inline .github/workflows/build.yml:8]: blank line

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
```
Comment (philburk) [inline .github/workflows/build.yml:5]: main or master

**Review by philburk — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (ggfan) [reply .github/workflows/build.yml:21]: fixed, thank you!

File: .github/workflows/build.yml (hunk 2 above)
Comment (ggfan) [reply .github/workflows/build.yml:8]: removed, left one blank line to separate from the next group of the commands

File: .github/workflows/build.yml (hunk 3 above)
Comment (ggfan) [reply .github/workflows/build.yml:5]: added master to it (future compatibility :-)). Thank you!

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        chmod +x gradlew
+        ./gradlew -q clean bundleDebug
+        popd
+    - name: build OboeTester
+      run: |
+        pushd apps/OboeTester
+        chmod +x gradlew
+        ./gradlew -q clean bundledDebug
+        popd
+# add future apps here
```
Comment (dturner) [inline .github/workflows/build.yml:31]: Please could we add the FX Lab app under "apps/fxlab" as well?

**Comment by dturner** [comment]
Looks great @ggfan, just one request. Thanks for doing this.

File: .github/workflows/build.yml (hunk 4 above)
Comment (ggfan) [reply .github/workflows/build.yml:38]: Done, thanks!

**Review by ggfan — COMMENTED** [review]
addressed the feedbacks, will merge it.

**Pull request merged**